README.md

```diff
@@ -11,17 +11,34 @@ This package includes the following functionality:
 
 * Display information about your current go installation, by running `go version` and `go env`
 * Format your code with `gofmt`, `goimports`, or `goreturns`; optionally run one of these tools on save of any `.go` file
+* Run `go install .` and `go test -c -o {tempdir} .` to verify your code can compile and to keep `gocode` suggestions up to date
 * Run tests, display test output, and display test coverage using `go test -coverprofile`
+* Display documentation for identifiers in source code using
+  [`gogetdoc`](https://github.com/zmb3/gogetdoc)
+* Rename the symbol under your cursor using `gorename`
 
 This package adds extra functionality by installing the following additional packages:
 
 * [autocomplete-go](https://atom.io/packages/autocomplete-go): Autocomplete using `gocode`
-* [builder-go](https://atom.io/packages/builder-go): Run `go install .` and `go test -c -o {tempdir} .` to verify your code can compile and to keep gocode suggestions up to date
```

Comment:
Clean up: only `go-debug` will be installed, everything else is already part of this package.